frontend/src/pages/ContactUs/ContactUs.jsx

```diff
@@ -3,9 +3,18 @@ import { Button2 } from "../../components/util/Button/index";
 import Recaptcha from "react-recaptcha";
 import style from "./ContactUs.module.scss";
 import Joi from "joi-browser";
+import { END_POINT } from "../../config/api";
+import { SimpleToast } from "../../components/util/Toast/index";
+import Loader from "../../components/util/Loader";
 
 export const ContactUs = (props) => {
-  const [isverfied, verified] = useState(false);
+  const [isverified, verified] = useState(false);
```

Comment:
Please use camelCase in here `isVerified` instead of `isverified`

Reply:
I can't because it is exported by that name only and changing it, I'll then have to make changes everywhere where it is used.